AstSemantics.md

```diff
@@ -154,6 +151,28 @@ by [available resources](Nondeterminism.md)). In the future, to support
 [>4GiB linear memory](FutureFeatures.md#heaps-bigger-than-4gib), support for
 indices with type `int64` will be added.
 
+### Adjusting memory size and permissions: `mmap`
+
+In the MVP the size of linear memory is fixed: The initial size of linear
+memory will remain unchanged for the life of that WebAssembly module. Later,
```

Comment:
What I suggested in #227 for "removing magic" was that it's specified by developer's code, typically in `_start`, using `mmap`. We can heavily restrict `mmap` so that it's not super capable in MVP, but I'd really like to avoid magical number in the binary that wasm implementations have to effectively translate to `mmap` for the rest of time. That gets weird when we do add `mmap` later!